.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/env/bin/python"
+}
```

Comment:
We don't want this committed. We should probably add `.vscode` to our `.gitignore` file and remove this from the index.